lib/nexmo_developer/app/presenters/page_title.rb

```diff
@@ -1,5 +1,7 @@
 class PageTitle
-  def initialize(product, document_title, default_title)
+  CONFIG = YAML.safe_load(File.open("#{Rails.configuration.docs_base_path}/config/header_meta.yml"))
```

Comment:
@benhayehudi can you add a method that returns the title from the config and remove the constant? 
Also, drop the new argument from the initializer

Reply:
✅